client/js/render.js

```diff
@@ -145,22 +145,10 @@ function renderChannelMessages(data) {
 
 function renderChannelUsers(data) {
 	const users = chat.find("#chan-" + data.id).find(".users");
-	let nicks = users.data("nicks") || [];
-	const oldSortOrder = {};
-
-	for (const i in nicks) {
-		oldSortOrder[nicks[i]] = i;
-	}
-
-	nicks = [];
-
-	for (const i in data.users) {
-		nicks.push(data.users[i].nick);
-	}
-
-	nicks = nicks.sort(function(a, b) {
-		return (oldSortOrder[a] || Number.MAX_VALUE) - (oldSortOrder[b] || Number.MAX_VALUE);
-	});
+	const nicks = data.users
+		.concat()
+		.sort((a, b) => b.lastMessage - a.lastMessage)
```

Comment:
I think I must have missed the point of renderChannelUsers. Is it not for the users for the sidebar? Because it looks like we are sorting based on when they last sent a message, which wouldn't make sense for the sidebar. So is this for the autocompletion?